transform/operations.go

```diff
@@ -311,6 +313,25 @@ func (c *toCamelCase) transform(raw interface{}) (interface{}, error) {
 	return strings.Join(arr, ""), nil
 }
 
+// removeHTML is a transformOperation which removes all html from a string.
+type removeHTML struct {
+	args map[string]string
+}
+
+func (c *removeHTML) init(args map[string]string) error {
+	return nil
+}
+
+func (c *removeHTML) transform(raw interface{}) (interface{}, error) {
```

Comment:
why not use a function that always accepts a string instead of convert the interface{} to string?

Reply:
That's how it's defined in the interface: https://github.com/GannettDigital/jstransform/blob/9da0e360189861fb47c1bd257367473864203ade/transform/transform.go#L24. Has to accept many different types, not just string.